bylaws.tex

```diff
@@ -164,32 +164,32 @@
 \bsubsection{The Introductory Packet}
 Each Introductory Process participant, after the first week, is given two (2) weeks to return a document that should contain the following:
 \begin{itemize}
-	\item A signature from all Active/Introductory Resident members, each Executive Board member, and fifteen (15) of any combination of the following:
+	\item A signature from all Active/Introductory Resident members, each Executive Board member, and fifteen (15) of any combination of the following:.
 	\begin{itemize}
-		\item Non-Resident members who have passed a Membership Evaluation
-		\item Alumni members (not including Non-Resident Executive Board members)
-		\item Honorary members
-		\item Advisory members
+		\item Non-Resident members who have passed a Membership Evaluation.
+		\item Alumni members (not including Non-Resident Executive Board members).
+		\item Honorary members.
+		\item Advisory members.
 	\end{itemize}
-	\item A description of each Executive Board position, including:
+	\item A description of each Executive Board position, including:.
```

Comment:
This is another example of an issue with the script

Reply:
ok, so remove all periods?

Reply:
I think that's up to discussion. I personally don't see this as an issue with the constituion. But at the very least we shouldn't have periods on the same lines as `:`

Reply:
Ok, I am doing this because my options were this or to fix a semantics problem that has been up since april. I didn't feel comfortable doing that, and figured that this was a low effect type of situation